src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets

```diff
@@ -23,5 +23,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- Set 7.0 as the TargetPlatformVersion for windows. -->
   <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'windows' and '$(TargetPlatformVersion)' == ''" >
     <TargetPlatformVersion>7.0</TargetPlatformVersion>
+    <TargetPlatformMoniker>Windows,Version=7.0</TargetPlatformMoniker>
+    <TargetPlatformDisplayName>Windows 7.0</TargetPlatformDisplayName>
```

Comment:
Do we need to set these?  If so, don't they need to be set if the TargetPlatformVersion is specified?  IE what will the `TargetPlatformMoniker` and `TargetPlatformDisplayName` be if the `TargetFramework` is `net5.0-windows10.0.19041`?

Reply:
I've included [test coverage](https://github.com/dotnet/sdk/pull/12202/files#diff-f9faa4d119d988118a9ff5cb495f33c1R61) for the TargetPlatformMoniker and TargetPlatformDisplayName properties in this PR. When the platform version is specified these properties are generated automatically but that generation occurs before this file is imported, so they aren't defined in this specific case.

Reply:
Makes sense.  I'd suggest explaining that in a comment here.